lua/octo/pickers/telescope/entry_maker.lua

```diff
@@ -694,6 +704,64 @@ function M.gen_from_octo_actions()
   end
 end
 
+function M.gen_from_notification()
+  local make_display = function(entry)
+    if not entry then
+      return nil
+    end
+
+    local columns = {
+      entry.obj.unread == true and icons.notification[entry.kind].unread or icons.notification[entry.kind].read,
+      { "#" .. (entry.obj.subject.url:match "/(%d+)$" or "NA") },
+      { string.sub(entry.obj.repository.full_name, 1, 50), "TelescopeResultsNumber" },
+      { string.sub(entry.obj.subject.title, 1, 100) },
+    }
+
+    local displayer = entry_display.create {
+      separator = " ",
+      items = {
+        { width = 2 },
+        { width = 12 },
```

Comment:
Let's make this smaller.  For instance, search has a max value of 6

https://github.com/wd60622/octo.nvim/blob/b4f00b00dec14fd396be979b04453336d6e83182/lua/octo/pickers/telescope/provider.lua#L541-L546

Reply:
An alternative is to make this dynamic sized like with the search as well. The width would be determined then passed to the entry_maker.